workspace/utilities/lib/ninja-img.xsl

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+	xmlns:exsl="http://exslt.org/common"
+	xmlns:str="http://exslt.org/strings"
+	xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
+
+	<xsl:template match="*[name() != 'img']" mode="ninja-img">
+		<xsl:element name="{name()}">
+			<xsl:apply-templates select="* | @* | text()" mode="ninja-img" />
+		</xsl:element>
+	</xsl:template>
+
+	<xsl:template match="img" mode="ninja-img">
+		<xsl:variable name="src" select="@src" />
+		<xsl:variable name="is-gif" select="contains($src, '.gif')" />
+		<xsl:variable name="is-svg" select="contains($src, '.svg')" />
+		<xsl:variable name="is-local" select="contains($src, $root)" />
+		<xsl:choose>
+			<xsl:when test="$is-local and not($is-gif) and not($is-svg)">
+				
+				<xsl:variable name="new-path" select="substring-after($src, 'workspace/')" />
+
+				<xsl:element name="{name()}">
```

Comment:
We should call the image xslt template if we can

Reply:
not sure for that... why?